engine/src/main/java/com/datatorrent/stram/client/RecordingsAgent.java

```diff
@@ -640,7 +640,7 @@ public JSONObject process(WebResource.Builder webResource, Class<JSONObject> cla
           }
       );
     } catch (Exception ex) {
-      LOG.error("Exception caught", ex);
+      LOG.error("Cannot start recording: ", ex);
```

Comment:
@gauravgopi123 That's because this particular one is actually an error we want the user to see. In this case, the user initiates a tuple recording but it fails. The other ones do not have this severity.